packages/docs/pages/blog/router-v6.mdx

```diff
@@ -0,0 +1,161 @@
+import { Alert } from '@patternfly/react-core'
+
+# Router v6 upgrade
+
+The router package [react-router-dom](https://reactrouter.com/en/main) has undergone a lot of changes in it lates v6 major release. Frontend applications in HCC are on a v5 which is no longer the LTS. Migration to V6 comes with some challenges. Mainly a router context can no longer be nested.
+
+This means our applications have to adapt to the new changes.
+
+## What are immediate actions?
+
+<Alert isInline variant="danger" title="Do not start migrating to router v6!">
+  The chrome UI has to be prepared first. Any application will have to adjust to chrome changes as well.
+</Alert>
+
+### My application has marked react-router-dom dependency as singleton
+
+If your application is on v5.x.x and has the router dependency marked as singleton, it will break once we upgrade chrome to v6. Please make sure this is not the case. This can only happen if you are using custom config, or have explicitly set the router dependency as shaped in your config.
```

Comment:
```suggestion
If your application is on v5.x.x and has the router dependency marked as singleton, it will break once we upgrade chrome to v6. Please make sure this is not the case. This can only happen if you are using custom config, or have explicitly set the router dependency as shaped in your config.

Let us (**@crc-experience-team** [#forum-consoledot-ui](https://app.slack.com/client/T027F3GAJ/C023VGW21NU)) so we can keep track of your application and please upgrade your react-router-dom to v6.
```